app/src/components/ShipmentForm/DetailsForm.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+	import { createForm } from 'felte';
+	import { reporter, ValidationMessage } from '@felte/reporter-svelte';
+	import Button from '../Buttons/Button.svelte';
+	import Details from './Details.svelte';
+	import { validator } from '@felte/validator-yup';
+	import * as yup from 'yup';
+	import type { DetailsFormInterface } from './interfaces';
+
+	export let initialValues: DetailsFormInterface;
+	export let onSubmit;
+	export let onBack;
+
+	export let showModal = true;
+
+	const schema = yup.object({
```

Comment:
You can move all schemas to the schema folder. Makes the code clearer.

Reply:
moved